lib/services/local/usertoken.go

```diff
@@ -145,6 +175,8 @@ func (s *IdentityService) UpsertUserTokenSecrets(ctx context.Context, secrets ty
 }
 
 const (
-	userTokenPrefix = "usertoken"
-	secretsPrefix   = "secrets"
+	// DELETE IN 9.0.0 in favor of userTokenPrefix.
+	passwordTokensPrefix = "resetpasswordtokens"
```

Comment:
Suggestion: rename to `legacyPasswordTokensPrefix`?

Reply:
revised